internal/builtinfunctions/functions.go

```diff
@@ -529,6 +532,37 @@ func getReadFileFunction() schema.CallableFunction {
 	return funcSchema
 }
 
+func getReadEnvVarFunction() schema.CallableFunction {
+	funcSchema, err := schema.NewCallableFunction(
+		"readEnvVar",
+		[]schema.Type{
+			schema.NewStringSchema(nil, nil, nil), // env var key name
+			schema.NewStringSchema(nil, nil, nil), // default value
+		},
+		schema.NewStringSchema(nil, nil, nil),
+		false,
+		schema.NewDisplayValue(
+			schema.PointerTo("readFile"),
+			schema.PointerTo(
+				"Returns the value of a system environment variable.\n"+
+					"Param 1: The key name of the environment variable.\n"+
```

Comment:
We often think of the environment variable name as the "key" in a key/value pair, and Python really leans into that by exposing the environment as a dict ... but I think in documentation just "environment variable name" would be better.

Reply:
Changed.